lua/lazyvim/plugins/extras/lang/rust.lua

```diff
@@ -17,9 +17,12 @@ return {
     ---@param opts cmp.ConfigSchema
     opts = function(_, opts)
       local cmp = require("cmp")
-      opts.sources = cmp.config.sources(vim.list_extend(opts.sources, {
-        { name = "crates" },
-      }))
+      opts.sources = vim.list_extend(
+        opts.sources or {},
+        cmp.config.sources({
+          { name = "crates" },
+        })
+      )
```

Comment:
This should actually just be:

```lua
      table.insert(opts.sources, { name = "crates" })
```

That will automatically set grouop_index to 1

Reply:
Updated. Thanks for reviewing :)